dist/dist-linux/pom.xml

```diff
@@ -14,7 +14,7 @@
   <dependencies>
     <dependency>
       <groupId>com.intel.analytics.bigdl.core.native.mkl</groupId>
```

Comment:
Do we need to change package name?

Reply:
yes, change the `groupId`

Reply:
groupid better to be unified as bigdl  -> com.intel.analytics.bigdl
artifactId can be bigdl-core-math-x86_64-linux, bigdl-core-math-mac, etc.

Reply:
changed artifact ID `com.intel.analytics.bigdl.core.native.mkl` to `com.intel.analytics.bigdl.core.native.math`
https://oss.sonatype.org/content/repositories/snapshots/com/intel/analytics/bigdl/core/native/